README.md

```diff
@@ -1,16 +1,121 @@
-# Geta.AutoMapper
+# Geta Mapping
+
+Geta Mapping is a project for common mapping logic. It consists of two libraries: [Geta.Mapping](#getamapping) and [Geta.AutoMapper](#getaautomapper).
+
+## Geta.Mapping
+
+### Description
+
+Geta.Mapping is a library with abstractions for common mapping logic.
+
+### Installation
+
+```
+Install-Package Geta.Mapping
+```
+
+### DI Configuration
+
+For `StructureMap` or `Lamar` configure interfaces to automatically connect to implementations:
+
+```csharp
+Scan(x =>
+{
+    x.TheCallingAssembly();
+    x.WithDefaultConventions();
+    x.ConnectImplementationsToTypesClosing(typeof(IMapper<,>));
+    x.ConnectImplementationsToTypesClosing(typeof(ICreateFrom<,>));
+});
+```
+
+### Usage
+
+Then create a mapping class you want to map one object to another. Inherit from `IMapper<TFrom, TTo>`.
+
+```csharp
+public class MyPocoToMyDtoMapper : IMapper<MyPoco, MyDto>
+{
+    public virtual void Map(MyPoco from, MyDto to)
+    {
+        to.Name = from.Name;
+    }
+}
+```
+
+This mapping implementation will work for any classes, even for those that has no parameter-less constructor.
+
+If your destination class has a parameter-less constructor, then you can implement `Mapper<TFrom, TTo>`.
+
+```csharp
+public class MyPocoToMyDtoMapper : Mapper<MyPoco, MyDto>
+{
+    public override void Map(MyPoco from, MyDto to)
+    {
+        to.Name = from.Name;
+    }
+}
+```
+
+Now you can use these mapper by injecting it.
+
+When you want to map one object to other, then use `IMapper<TFrom, TTo>` interface.
```

Comment:
"...one object to other (if have already instances of these classes)"

Reply:
I think additional comments not necessary here. The object is already an instance of the class.